src/lib/tabs/tab-body.html

```diff
@@ -1,4 +1,6 @@
 <div class="mat-tab-body-content" #content
+     (swipe)="onSwipe.emit($event)"
+     [style.touch-action]="'pan-y'"
```

Comment:
Add a comment for why this needs to be an inline style? Also, since this is static, you don't need a binding; just `style="touch-action: pan-y"` is good.